plugins/modules/sonic_ospfv3_interfaces.py

```diff
@@ -0,0 +1,599 @@
+
+#############################################
+#                WARNING                    #
+#############################################
+#
+# This file is auto generated by the resource
+#   module builder playbook.
+#
+# Do not edit this file manually.
+#
+# Changes to this file will be over written
+#   by the resource module builder.
+#
+# Changes should be made in the model used to
+#   generate this file or in the resource module
+#   builder template.
+#
+#############################################
+
+"""
+The module file for sonic_ospfv3_interfaces
+"""
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = """
+---
+module: sonic_ospfv3_interfaces
+version_added: '3.1.0'
+short_description: Configure OSPFv3 interface mode protocol settings on SONiC.
+description:
+  - This module provides configuration management of OSPFv3 interface mode parameters on devices running SONiC.
+  - Configure VRF instance before configuring OSPFv3 in a VRF.
+  - Configure OSPFv3 instance before configuring OSPFv3 in interfaces.
+author: "Thenmozhi Gopal (@thenmozhi-gopal), Mansi Jharia (@Mansi062001)"
+options:
+      config:
+        description:
+          - Specifies the OSPFv3 interface mode related configuration.
+        type: list
+        elements: dict
+        suboptions:
+          name:
+            required: True
+            type: str
+            description:
+              - Full name of the interface, i.e. Ethernet1.
+          advertise:
+            description:
+              - Enable OSPFv3 interface advertise.
+              - expects name of a prefix list.
+            type: str
+          area_id:
+            description:
+              - OSPFv3 Area ID of the network (A.B.C.D or 0 to 4294967295).
+            type: str
+          cost:
+            description:
+              - Configure OSPFv3 interface cost (1 to 65535).
+            type: int
+          dead_interval:
+            description:
+              - Configure OSPFv3 adjacency dead interval (1 to 65535).
+            type: int
+          hello_interval:
+            description:
+              - Configure OSPFv3 neighbour hello interval (1 to 65535).
+            type: int
+          mtu_ignore:
+            description:
+              - Disable OSPFv3 MTU mismatch detection.
+            type: bool
+          passive:
+            description:
+              - Configure ospfv3 interface as passive.
+            type: bool
+          priority:
+            description:
+              - Configure OSPFv3 adjacency router priority (0 to 255).
+            type: int
+          retransmit_interval:
+            description:
+              - Configure OSPFv3 retransmit interval (2 to 65535).
+            type: int
+          transmit_delay:
+            description:
+              - Configure OSPFv3 transmit delay (1 to 65535).
+            type: int
+          bfd:
```

Comment:
move bfd next to area_id for alphabetical order and arrange all attributes in alphabetical order.

Reply:
so does bfd_profile before enable.